Dockerfile

```
@@ -0,0 +1,27 @@
+FROM python:3 as base
+RUN mkdir /opt/todoapp
+RUN chmod 775 /opt/todoapp
+WORKDIR /opt/todoapp
+COPY . .
+#COPY todo_app/* todo_app/
+RUN pip install poetry 
+RUN pip install --no-cache-dir pyproject.toml
+RUN pip install gunicorn flask 
```

Given that we've installed poetry, we might as well use it

```suggestion
RUN pip install poetry 
RUN poetry install
```